controllers/storagecluster/placement_test.go

```diff
@@ -145,7 +145,7 @@ func TestGetPlacement(t *testing.T) {
 		},
 		{
 			label:          "Case 3: LabelSelector to modify the default Placements correctly",
-			storageCluster: mockStorageCluster,
+			storageCluster: mockStorageCluster.DeepCopy(),
```

Comment:
Why are these changes required?

Reply:
These changes are not the part of this PR.  The unit-tests were broken, because of which, I was not able to test the PR. Once the PR: https://github.com/openshift/ocs-operator/pull/1306 is merged, these changes will no longer be relevant.